cmake/ci.cmake

```diff
@@ -434,21 +434,21 @@ add_custom_target(ci_test_clang
 
 foreach(CXX_STANDARD 11 14 17 20)
     add_custom_target(ci_test_gcc_cxx${CXX_STANDARD}
-        COMMAND CXX=${GCC_TOOL} ${CMAKE_COMMAND}
+        COMMAND CXX=${GCC_TOOL} CXXFLAGS="${GCC_CXXFLAGS}" ${CMAKE_COMMAND}
             -DCMAKE_BUILD_TYPE=Debug -GNinja
-            -DCMAKE_CXX_STANDARD=${CXX_STANDARD} -DCMAKE_CXX_STANDARD_REQUIRED=ON
             -DJSON_BuildTests=ON -DJSON_FastTests=ON
+            -DJSON_TestStandards=${CXX_STANDARD}
             -S${PROJECT_SOURCE_DIR} -B${PROJECT_BINARY_DIR}/build_gcc_cxx${CXX_STANDARD}
         COMMAND ${CMAKE_COMMAND} --build ${PROJECT_BINARY_DIR}/build_gcc_cxx${CXX_STANDARD}
         COMMAND cd ${PROJECT_BINARY_DIR}/build_gcc_cxx${CXX_STANDARD} && ${CMAKE_CTEST_COMMAND} --parallel ${N} --output-on-failure
         COMMENT "Compile and test with GCC for C++${CXX_STANDARD}"
     )
 
     add_custom_target(ci_test_clang_cxx${CXX_STANDARD}
-        COMMAND CXX=${CLANG_TOOL} ${CMAKE_COMMAND}
+        COMMAND CXX=${CLANG_TOOL} CXXFLAGS="${CLANG_CXXFLAGS}" ${CMAKE_COMMAND}
             -DCMAKE_BUILD_TYPE=Debug -GNinja
-            -DCMAKE_CXX_STANDARD=${CXX_STANDARD} -DCMAKE_CXX_STANDARD_REQUIRED=ON
-            -DJSON_BuildTests=ON
+            -DJSON_BuildTests=ON -DJSON_FastTests=ON
```

Comment:
Why are `DJSON_FastTests` set to `ON` here?

Reply:
While an unintentional copy&paste mistake, I left it in for parity with ci_test_gcc_cxx above. Of course we could go the other direction but I believe the coverage provided by other tests (e.g., ci_test_gcc & ci_test_clang) should be adequate, no?

Reply:
No, it's fine. Thanks for the quick response.